core/common/assets/js/api/core/commands.js

```diff
@@ -269,38 +283,139 @@ export default class Commands extends CommandsBackwardsCompatibility {
 	 * @returns {boolean|*} results
 	 */
 	run( command, args = {} ) {
-		if ( ! this.beforeRun( command, args ) ) {
+		if ( ! this.validateRun( command, args ) ) {
 			return false;
 		}
 
-		const component = this.getComponent( command ),
-			container = component.getRootContainer();
+		this.beforeRun( command, args );
```

Comment:
`if ( ! this.beforeRun( command, args ) ) {`

Reply:
validateRun is now used also in routes.